src/client/components/switcherSound/switcherSound.css

```diff
@@ -5,26 +5,29 @@
     display: flex;
     width: 100%;
     max-width: 1024px;
-    margin: 0 auto;
-    padding: 0 25px;
+    justify-content: flex-end;
     color: white;
```

Comment:
take this color from variables, same in trackDurat.css

Reply:
done